corehq/apps/locations/fixtures.py

```diff
@@ -237,18 +237,18 @@ def _get_include_without_expanding_locations(domain, location_type):
     """
     include_without_expanding = location_type.include_without_expanding
     if include_without_expanding is not None:
-        # add in other "forced" locations the user should have, along with their ancestors
-        forced_locations = set(SQLLocation.active_objects.filter(
-            domain__exact=domain,
-            location_type=include_without_expanding
-        ))
-        forced_ancestors = {
-            ancestor
-            for location in forced_locations
-            for ancestor in location.get_ancestors()
-        }
-
-        return forced_locations | forced_ancestors
+        forced_location_level = set(
+            SQLLocation.active_objects.
+            filter(domain__exact=domain, location_type=include_without_expanding).
+            values_list('level', flat=True)
+        ) or None
```

Comment:
This'll return the level value for each location of that type - you could instead use a `distinct` query to get only the unique values (of which there should only be one).  Actually you could also probably  just slap a `.first()` on the end of what you've got here and use that, since we expect them all to be the same anyways.